kuma/dashboards/jinja2/dashboards/includes/revision_dashboard_body.html

```diff
@@ -92,6 +94,8 @@
 
                         {% endif %}
                     </td>
+                    {% else %}
+                    <td class="dashboard-spam">&nbsp;</td>
```

Comment:
This as well as the entry on line 14 above is causing the automated tests to fail, as it expects and empty array to be returned:

```
page = pq(response.content)
spam_table_cell = page.find('td.dashboard-spam')
eq_(spam_table_cell, [])
```

This is in kuma/dashboards/tests/test_views.py on line 85-87

I am guessing you added this to overcome the rows breaking out of it's container so, one would either need to update the test or, explore another solution, perhaps a different class name or no class name at all? @stephaniehobson Thoughts?